packages/keystone/src/admin-ui/system/createAdminMeta.ts

```diff
@@ -16,8 +16,17 @@ export function createAdminMeta(
     views: [],
   };
 
+  const omittedLists: string[] = [];
+
   for (const [key, list] of Object.entries(initialisedLists)) {
     const listConfig = lists[key];
+    if (list.graphql.isEnabled.query === false) {
```

Comment:
is `isEnabled.query` the only flag we want to skip on here?

Reply:
As a separate thing, if `list.graphql.isEnabled.create`/`list.graphql.isEnabled.delete` is false, we should force hideCreate/hideDelete to return false https://github.com/keystonejs/keystone/blob/3cfc2a3839142dd3ccdbf1dd86768257e9acc0dc/packages/keystone/src/admin-ui/system/getAdminMetaSchema.ts#L204-L227. We should also force the itemView fieldMode to read for all the fields of a list when the list has update omitted so you can't get a confusing error by editing and then trying to save (doing that is in the same file as above)

While we could technically make the creating in the Admin UI work for a list that has querying disabled so arguably the above check should be `isEnabled.type`, we definitely shouldn't do that now so `isEnabled.query` is correct imo.

Reply:
Yes.

Reply:
> As a separate thing, if `list.graphql.isEnabled.create`/`list.graphql.isEnabled.delete` is false, we should force hideCreate/hideDelete to return false
> 
> https://github.com/keystonejs/keystone/blob/3cfc2a3839142dd3ccdbf1dd86768257e9acc0dc/packages/keystone/src/admin-ui/system/getAdminMetaSchema.ts#L204-L227
> 
> . We should also force the itemView fieldMode to read for all the fields of a list when the list has update omitted so you can't get a confusing error by editing and then trying to save (doing that is in the same file as above)
> While we could technically make the creating in the Admin UI work for a list that has querying disabled so arguably the above check should be `isEnabled.type`, we definitely shouldn't do that now so `isEnabled.query` is correct imo.

As discussed will implement this in a separate PR